app/src/main/res/layout/activity_friends.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
There's some inconsistent spacing in the layout files that would be good to clean up